stubs/pandas/__init__.py

```
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import datetime as _dt
+import types
+
+NA = object()
+NaT = object()
+
+class Series:  # pragma: no cover - simple stub
+    """Placeholder for pandas.Series"""
+
+    def __init__(self, *args, **kwargs) -> None:  # noqa: D401
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `D401`)

Remove unused `noqa` directive

(RUF100)